person.rb

```diff
@@ -27,6 +28,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
Great job @dvcorreia1984 🤝

- [ ] As stated in the project requirements, kindly ensure the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
![add-rental](https://github.com/dvcorreia1984/oop-school-library/assets/63917725/3728e52c-d7ba-4214-92c2-df17aa0149c2)

- [ ] For the Student class, kindly ensure there's a method that serves as the 'belongs-to' side (a student belongs to a classroom)`. You may take a look at this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more clarity.
![studentrb](https://github.com/dvcorreia1984/oop-school-library/assets/63917725/4c717aca-64be-4643-a7dc-a0efb4ea88aa)